docs/source/llms/langchain/index.rst

```diff
@@ -240,3 +240,89 @@ How can I use a streaming API with LangChain?
     As of the MLflow 2.12.2 release, LangChain models that support streaming responses that have been saved using MLflow 2.12.2 (or higher) can be loaded and used for 
     streamable inference using the ``predict_stream`` API. Ensure that you are consuming the return type correctly, as the return from these models is a ``Generator`` object.
     To learn more, refer to the `predict_stream guide <https://mlflow.org/docs/latest/models.html#how-to-load-and-score-python-function-models>`_.
+
```

Comment:
What about adding a quick prompt in `I can't save my chain, agent, or retriever with MLflow.` section, like "try model-from-code feature if you face issues when logging models"?